sync/class.jetpack-sync-module-posts.php

```diff
@@ -113,6 +132,9 @@ function filter_post_content_and_add_links( $post_object ) {
 			return $blocked_post;
 		}
 
+		// lets not do oembed just yet.
+		$this->remove_embed();
```

Comment:
Should we also do this for comments? Not sure of oembed is supported for comments.